<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>491</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Auto Files Bkup</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>113</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select path</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit_select_path</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_select_path">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>110</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_path">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>109</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_backup_path">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>149</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_backup_path">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>148</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>153</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Backup path</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit_backup_path</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start_backup">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 31, 157);
color: #fefefe;</string>
    </property>
    <property name="text">
     <string>Start backup</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_delay">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>187</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>191</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay (minutes)</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit_delay</cstring>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_backup_status">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0.0);</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK JP Black</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Auto Files Bkup</string>
    </property>
   </widget>
   <zorder>lineEdit_select_path</zorder>
   <zorder>pushButton_select_path</zorder>
   <zorder>lineEdit_backup_path</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_backup_path</zorder>
   <zorder>pushButton_start_backup</zorder>
   <zorder>lineEdit_delay</zorder>
   <zorder>label_3</zorder>
   <zorder>textBrowser_backup_status</zorder>
   <zorder>line</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_select_path</tabstop>
  <tabstop>pushButton_select_path</tabstop>
  <tabstop>lineEdit_backup_path</tabstop>
  <tabstop>pushButton_backup_path</tabstop>
  <tabstop>lineEdit_delay</tabstop>
  <tabstop>pushButton_start_backup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
